Mathlib/Combinatorics/SimpleGraph/Maps.lean

```diff
@@ -219,6 +219,10 @@ theorem induce_spanningCoe {s : Set V} {G : SimpleGraph s} : G.spanningCoe.induc
 theorem spanningCoe_induce_le (s : Set V) : (G.induce s).spanningCoe ≤ G :=
   map_comap_le _ _
 
+lemma spanningCoe_induce_adj {s : Set V} {u v : s} :
+    (G.induce s).spanningCoe.Adj u v ↔ G.Adj u v := by
+  simp
```

Comment:
This lemma is weird:
1. You should generalise it to 
```suggestion
lemma spanningCoe_induce_adj {s : Set V} {u v : V} :
    (G.induce s).spanningCoe.Adj u v ↔ u ∈ s ∧ v ∈ s ∧ G.Adj u v := by
  simp
```
2. It's proved by simp, so you should drop it